src/index.html

```diff
@@ -10,8 +12,155 @@
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      >
+
+      <div class="container">
+
+        <div class="describe">Personal information</div>
+      
+        <div class="form-field">
+          Surname:
+          <input type="text"
+          name="surname"
+          autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          Name:
+          <input type="text"
+          name="name"
+          autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          How old are You?
+          <input type="number"
+          name="yearsold"
+          value="12"
+          minlength="1"
+          maxlength="100"
+          >
+        </div>
+
+        <div class="form-field">
+          Full date of birth:
+          <input type="date"
+          name="fullbirth"
+          >
+        </div>
+         
+        <div class="form-field">
+          I accept the term of the agreement
+          <input type="checkbox">
+        </div>
+      </div>
+
+      <div class="container">
+
+        <div class="describe">
+          Registration
+        </div>
+
+        <div class="form-field"
+        >
+          E-mail:
+          <input type="email"
+          name="mail"
+          placeholder="email@example.com"
+          >
+        </div>
```

Comment:
use labels instead of divs, divs don't provide interaction with input, fix it everywhere
```suggestion
        <label class="form-field">
          E-mail:
          <input 
            type="email"
            name="mail"
            placeholder="email@example.com"
          >
        </label>
```